core/src/epicli/data/common/defaults/epiphany-cluster.yml

```diff
@@ -11,8 +11,10 @@ specification:
     key_path: /root/.ssh/epiphany-operations/id_rsa # YOUR-SSH-KEY-PATH
   cloud:
     subscription_name: YOUR-SUB-NAME
+    resource_group_name: YOUR-RESOURCE-GROUP-NAME
```

Comment:
shouldn't this be created based on cluster name, similar to this what we have on AWS? we also are not using this parameter on AWS in terraform templates probably?

Reply:
This is already configurable for AWS as well